lucene/CHANGES.txt

```diff
@@ -47,6 +47,13 @@ API Changes
 * LUCENE-9264: SimpleFSDirectory has been removed in favor of NIOFSDirectory.
   (Yannick Welsch)
 
+* LUCENE-9281: Use java.util.ServiceLoader to load codec components and analysis
+  factories to be compatible with Java Module System. This allows to load factories
+  without META-INF/service from a Java module exposing the factory in the module
+  descriptor. This breaks backwards compatibility as custom factories must now
+  also implement the default constructor (see MIGRATE.txt).
+  (Uwe Schindler, Dawid Weiss, Tomoko Uchida)
```

Comment:
i did nothing here and had no idea about this issue...

Reply:
Removed you, I just added you because you were doing a lot of preparatory work and also commented on some changes.

Reply:
Thanks... I just looked through the changes and approved (though i don't fully understand about it).